### PR TITLE
Fix Dashboard-Bloc-provider error and Code Cleanup

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
#### Summary of Changes (What does this PR do?)
- [x] Code cleanup
- [x]  Insert `Dashboard bloc` in Multi provider.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Run the app to test if it's running as expected.

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

File: mobile/lib/app_config.dart [hunk 1]
```diff
@@ -1,15 +1,9 @@
 import 'package:flutter/material.dart';
 
-// 1
 enum Environment { dev, prod }
 
-// 2
 class AppConfig extends InheritedWidget {
-  // 3
-  final Environment environment;
```
Comment (BenjaminSsempala) [inline mobile/lib/app_config.dart:9]: Code Cleanup

File: mobile/lib/main.dart [hunk 2]
```diff
@@ -2,10 +2,6 @@ import 'dart:io';
 
 import 'package:app/app_config.dart';
 import 'package:app/main_common.dart';
-
-import 'package:app/blocs/blocs.dart';
```
Comment (BenjaminSsempala) [inline mobile/lib/main.dart:6]: Removing Unused imports.

File: mobile/lib/main_common.dart [hunk 3]
```diff
@@ -9,10 +10,6 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:provider/provider.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
 
-import 'blocs/feedback/feedback_bloc.dart';
-import 'blocs/map/map_bloc.dart';
-import 'blocs/nearby_location/nearby_location_bloc.dart';
```
Comment (BenjaminSsempala) [inline mobile/lib/main_common.dart:14]: Removing duplicate imports.

File: mobile/lib/main_common.dart [hunk 4]
```diff
@@ -36,6 +33,9 @@ class AirQoApp extends StatelessWidget {
         BlocProvider(
           create: (BuildContext context) => MapBloc(),
         ),
+        BlocProvider(
+          create: (BuildContext context) => DashboardBloc(),
```
Comment (BenjaminSsempala) [inline mobile/lib/main_common.dart:37]: Adding Dashboard Bloc.

File: mobile/lib/widgets/dialogs.dart [hunk 5]
```diff
@@ -5,11 +5,8 @@ import 'package:flutter/material.dart';
 import 'package:flutter_svg/svg.dart';
 import 'package:geolocator/geolocator.dart';
 
-import '../blocs/nearby_location/nearby_location_bloc.dart';
```
Comment (BenjaminSsempala) [inline mobile/lib/widgets/dialogs.dart:8]: Removing unused imports.

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**